src/DotNetWorker.Core/FunctionsApplication.cs

```diff
@@ -67,6 +67,18 @@ public void LoadFunction(FunctionDefinition definition)
 
         public async Task InvokeFunctionAsync(FunctionContext context)
         {
+            // This will act as an internal activity that represents remote Host activity. This cannot be tracked as this is not associate to an ActivitySource. 
+            using Activity activity = new Activity(nameof(InvokeFunctionAsync));
```

Comment:
We have an existing `FunctionActivitySourceFactory` - how does that interact with OTel, if at all? Is it being removed or disabled when OTel is enabled?

Reply:
This is not in play be default. Users can still add a source and can consume this activity. it will work seamlessly.